api/src/Util/EntityMap.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Util;
+
+use App\Entity\BaseEntity;
+
+class EntityMap {
+    private $map = [];
+
+    public function add(BaseEntity $prototype, BaseEntity $entity) {
+        $key = $prototype::class.'#'.$prototype->getId();
```

Comment:
das # könnte man in eine konstante auslagern

Reply:
Oder wir könnten $iriConverter->getIriFromItem($prototype) verwenden. Dann wären wir konsistent mit anderen Entity-Lookups.

Reply:
$iriConverter heisst DI - nicht?. Scheint mir etwas übertrieben für sehr wenig Ertrag.
Aktuell ist EntityMap so autonom, dass diese beliebig konstruiert werden kann - z.B. auch in der Entity selber.